modules/tinymce/src/plugins/template/test/ts/browser/SelectedContentTest.ts

```diff
@@ -1,51 +1,49 @@
-import { Log, Mouse, Pipeline, UiFinder, Waiter } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { TinyApis, TinyLoader } from '@ephox/mcagar';
-import { SugarElement } from '@ephox/sugar';
-import TemplatePlugin from 'tinymce/plugins/template/Plugin';
-import SilverTheme from 'tinymce/themes/silver/Theme';
+import { afterEach, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/mcagar';
 
-UnitTest.asynctest('browser.tinymce.plugins.template.SelectedContentTest', (success, failure) => {
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/template/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
 
-  TemplatePlugin();
-  SilverTheme();
+import { insertTemplate } from '../module/InsertTemplate';
+import { Settings } from '../module/Settings';
 
-  TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-    const tinyApis = TinyApis(editor);
+describe('browser.tinymce.plugins.template.DatesTest', () => {
```

Comment:
```suggestion
describe('browser.tinymce.plugins.template.SelectedContentTest', () => {
```

Copy/pasted incorrectly I assume?